src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonValue.CreateOverloads.cs

```diff
@@ -14,175 +14,175 @@ public partial class JsonValue
         /// <param name="value">The underlying value of the new <see cref="JsonValue"/> instance.</param>
         /// <param name="options">Options to control the behavior.</param>
         /// <returns>The new instance of the <see cref="JsonValue"/> class that contains the specified value.</returns>
-        public static JsonValue Create(bool value, JsonNodeOptions? options = null) => new JsonValuePrimitive<bool>(value, JsonMetadataServices.BooleanConverter);
+        public static JsonValue Create(bool value, JsonNodeOptions? options = null) => new JsonValuePrimitive<bool>(value, JsonMetadataServices.BooleanConverter, options);
```

Comment:
This was a bug where options were not being passed, right? Should we add a test?

Reply:
It was probably done intentionally since the only exposed option is `PropertyNameCaseInsensitive` which does not apply to primitives. As such, there's no discernible difference in observable behavior. That being said, it is always a good idea to pass on the options because it might receive additional capabilities in the future.